app/assets/javascripts/discourse/app/widgets/search-menu-results.js

```diff
@@ -702,3 +710,40 @@ createWidget("random-quick-tip", {
     }
   },
 });
+
+createWidget("search-menu-recent-searches", {
+  tagName: "div.search-menu-recent",
+
+  html() {
+    const content = [];
+    content.push(
```

Comment:
This is not super important but you might be able to use the `hbs` helper for a template like this.

Reply:
Oh yes, that looks much better with the `hbs` helper.